.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Don't forget a short, descriptive title! -->
+<!-- This is a template, feel free to modify as your specific pull request
+needs, but keep the same basic structure. -->
+
+# What's in this PR?
+
+## Why was this created?
+
+## Changes Made
+
+### Reasons
+
+### Before
+
+### After
+
+## References
+Fixes #
+
+Progress on: #
```

Comment:
I would reduce this down. It feels a little overwhelming. I would condense down to just:

```
# What's in this PR?

List the changes you made and state your reasons for them.

Make sure any changes to code include changes to that code's documentation.
```

And keep the references section.